src/api/c/image.cpp

```diff
@@ -264,6 +264,28 @@ af_err af_is_window_closed(bool *out, const af_window wind)
 #endif
 }
 
+af_err af_set_visibility(const af_window wind, const bool is_visible)
+{
+#if defined(WITH_GRAPHICS)
+    if(wind==0) {
```

Comment:
He's asleep now and we have to release soon :( looks like we'll have to fix it at a later point.